docs/docs/quickstart.mdx

```diff
@@ -376,43 +405,49 @@ You can retrieve the email however by opening the MailSlurper UI at
 
 You should see something like this:
 
-<img
-  alt="User Email Verification"
-  src={useBaseUrl('img/docs/mailslurper-quickstart.png')}
-/>
+<p>
+  <img
+    alt="User Email Verification"
+    src={useBaseUrl('img/docs/mailslurper-quickstart.png')}
+  />
+</p>
 
 If not, hard refresh the tab or click on the home icon in the menu bar.
 
 Next, click the verification link. You will end up at the dashboard, with a
-verified E-Mail Address (check the `verified` and `verified_at` field in the
-JSON Payload):
+verified email address (check the `verified` and `verified_at` field in the
+JSON response):
 
-<img
-  alt="SecureApp Dashboard"
-  src={useBaseUrl('img/docs/secureapp-verified-dashboard.png')}
-/>
+<p>
+  <img
+    alt="SecureApp Dashboard"
+    src={useBaseUrl('img/docs/secureapp-verified-dashboard.png')}
+  />
+</p>
 
 To re-request the verification email, fill out the form at
 [127.0.0.1:4455/verify](http://127.0.0.1:4455/verify).
 
-#### Configuration Used
+#### Configuration
 
-You can find all configuration files for this quickstart guide in
-`./contrib/quickstart` and `./quickstart.yml`. To understand what each of those
-individual configuration files are doing, you must consult the other chapters of
+You can find all configuration files used for this quickstart guide in
+[`./contrib/quickstart/kratos`](https://github.com/ory/kratos/tree/v0.3.0-alpha.1/contrib/quickstart/kratos/email-password)
+, [`./quickstart.yml`](https://github.com/ory/kratos/blob/v0.3.0-alpha.1/quickstart.yml),
+and [`./quickstart-standalone.yml`](https://github.com/ory/kratos/blob/v0.3.0-alpha.1/quickstart-standalone.yml).
+To understand what each of those configuration files does, consult the other chapters of
 this documentation.
 
 :::note
 
-To get a minimal version of ORY Kratos running, you need to set configuration
-items `identity.traits.default_schema_url` and `dsn`. You should also configure
-`urls.*_ui` because your users will end up at fallbacks otherwise.
+To get a minimal version of Kratos running, you need to set configuration
+values for [`identity.traits.default_schema_url`](https://github.com/ory/kratos/blob/v0.3.0-alpha.1/contrib/quickstart/kratos/email-password/.kratos.yml#L67) and [`DSN`](https://github.com/ory/kratos/blob/v0.3.0-alpha.1/quickstart.yml#L42). You should also configure
```

Comment:
If we fix the tag here, we need to update it in our release pipeline so that it stays up to date with the newest version.

Reply:
True, but I was thinking if we specified a line number against `master` that the line might change in the future to be something other than what it's meant to refer to.

Reply:
Yes, that's true. So to keep it as part of our update pipeline I think that all you need to do is add quickstart.mdx here: https://github.com/ory/kratos/blob/master/docs/config.js#L20